egograph/backend/dependencies.py

```
@@ -119,3 +122,9 @@ def get_youtube_repository(
     config: BackendConfig = Depends(get_config),
 ) -> YouTubeRepository:
     return YouTubeRepository(_require_r2(config), tz=config.timezone)
+
+
+def get_google_health_repository(
+    config: BackendConfig = Depends(get_config),
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)